qiling/core.py

```diff
@@ -208,10 +208,26 @@ def shellcode(self):
 
 
     def run(self):
+        if self.gdb is not None:
+            try:
+                if self.gdb is True:
+                    self.gdbserver()
+                else:
+
+                        ip, port = self.gdb.split(':')
```

Comment:
incorrect indention and extra empty line :(

Reply:
oh，my bad

Reply:
will PR soon, thanks!